frontend/app/src/types/history/trade/location.ts

```diff
@@ -7,7 +7,9 @@ export const TradeLocation = z.enum([
   ...SUPPORTED_EXCHANGES,
   ...SUPPORTED_TRADE_LOCATIONS,
   'gitcoin',
-  'loopring'
+  'loopring',
+  'polygon pos',
+  'arbitrum one'
```

Comment:
these two are inside `SUPPORTED_TRADE_LOCATIONS`. So just change the hardcoded enum. I think it's the same with what I do https://github.com/rotki/rotki/pull/6676

Reply:
i've removed them, what is inside `SUPPORTED_TRADE_LOCATIONS` has underscores
`polygon_pos` `arbitrum_one`

Reply:
> i've removed them, what is inside `SUPPORTED_TRADE_LOCATIONS` has underscores `polygon_pos` `arbitrum_one`

yeah just remove the underscore (?). Because the correct location is without underscore. I think the `<LocationDisplay/>` still breaks

Reply:
Okay, thanks